private/buf/cmd/buf/command/dep/depupdate/depupdate.go

```diff
@@ -143,11 +143,11 @@ func run(
 	if err := workspaceDepManager.UpdateBufLockFile(ctx, configuredDepModuleKeys); err != nil {
 		return err
 	}
-	// Prune the buf.lock. This also verifies the workspace builds again.
-	// Building also has the side effect of doing tamper-proofing.
```

Comment:
This comment should remain and continue to be true.

However, based on the latest changes, it is no longer true. The workspace is no longer built.

Reply:
The workspace is still being built on the next line and then we are still logging out the unused configured deps for the user.

Reply:
`GetWorkspace` does not build the workspace.

Reply:
@doriable where does it get build? Previous prune codepath explicitly calls `controller.GetImageForWorkspace`

Reply:
ah, nm, right there in the diff, lol

Reply:
wait, no, that is just `GetWorkspace` not `GetImageForWorkspace`

Reply:
Correct - the previous code called `GetImageForWorkspace`, which does the build, this code does not

Reply:
Ah my apologies, I misconstrued that as compiling the `ModuleSet` rather than building the image. Yes, I should add that step back.